maven-plugin-report-plugin/pom.xml

```diff
@@ -88,6 +86,7 @@
     <dependency>
       <groupId>org.apache.maven.doxia</groupId>
       <artifactId>doxia-sink-api</artifactId>
+      <version>1.11.1</version>
```

Comment:
Most of dependency is managed, this one will not - only small inconsistency

Reply:
It is just a single one. I didn not create property on purpose.